noxconfig.py

```diff
@@ -11,12 +11,14 @@
 class Config:
     root: Path = Path(__file__).parent
     doc: Path = Path(__file__).parent / "doc"
+    source: Path = Path("exasol_integration_test_docker_environment")
     version_file: Path = (
         Path(__file__).parent
         / "exasol_integration_test_docker_environment"
         / "version.py"
     )
     path_filters: Iterable[str] = ("dist", ".eggs", "venv", "resources")
+    python_versions = ["3.9", "3.10", "3.11", "3.12", "3.13"]
```

Comment:
needed to explicitly add as sonar depends on this. added 3.13 to cover what's in the pyproject.toml, as we don't yet have an outward facing python_versions from PTB (except from the matrices)